<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OttoInputWindow</class>
 <widget class="QMainWindow" name="OttoInputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1019</width>
    <height>633</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1019</width>
    <height>633</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Thermodynamic Simulator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(79, 29, 11);
color: rgb(255,255,255);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Compression_Ratio_Label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(252, 140, 3)</string>
    </property>
    <property name="text">
     <string>Compression Ratio</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Expansion_Efficiency_Label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(252, 140, 3)</string>
    </property>
    <property name="text">
     <string>Expansion Efficiency</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Initial_Temperature_Label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(252, 140, 3)</string>
    </property>
    <property name="text">
     <string>Initial Temperature (K)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Compression_Efficiency_Label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(252, 140, 3)</string>
    </property>
    <property name="text">
     <string>Compression Efficiency</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Initial_Pressure_Label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(252, 140, 3)</string>
    </property>
    <property name="text">
     <string>Initial Pressure (kPa)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Expansion_eff_input">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>490</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(206, 88, 4)</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter the Expansion Efficiency</string>
    </property>
   </widget>
   <widget class="QLabel" name="Otto_Image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>521</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../graphics/otto_cycle.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1011</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Input Otto Cycle Properties:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Compression_eff_input">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(206, 88, 4)</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter the Compression Efficiency</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ratio_Input">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(206, 88, 4)</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter the Compression Ratio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Initial_pressure_input">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(206, 88, 4)</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter the Initial Pressure in kPa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="initial_temperature_input">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(206, 88, 4)</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter the Initial Temperature in Kelvins</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Help_Button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(152, 51, 0)</string>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Begin_Button">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>540</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(152, 51, 0)</string>
    </property>
    <property name="text">
     <string>Begin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back_Button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>540</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(152, 51, 0)</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
